packages/npm-lit/demo/index.ts

```diff
@@ -0,0 +1,69 @@
+import { atom, Atom, peek } from '@reatom/core'
+import { withReatomElement, watch } from '../src/index.js'
+import { LitElement, html } from 'lit'
+
+const timer = atom(0)
+const countAtom = atom(0)
+
+const increment = atom(() => {
+  countAtom(countAtom() + 1)
+})
+
+setInterval(() => {
+  timer(timer() + 1)
+}, 1_000)
+
+const RenderCountElement = withReatomElement(
+  class RenderCountElement extends LitElement {
```

Comment:
Why not as in https://github.com/lit/lit/blob/main/packages/labs/signals/README.md
```suggestion
class RenderCountElement extends ReatomWatcher(LitElement) {
```

Reply:
We need to replace render method of final user class. If we replate render of LitElement, it will dosen't work

Reply:
Please, add a note about this in the sources